scripts/add-chain.sh

```diff
@@ -34,6 +46,8 @@ public_rpc: $PUBLIC_RPC
 sequencer_rpc: $SEQUENCER_RPC
 explorer: $EXPLORER
 
+type: $TYPE
```

Comment:
Let's name this something more descriptive like `superchain_level` (name based on [this slide](https://docs.google.com/presentation/d/1IBHw_KR96DgRaGCvxMSlqMMA3FshbYap9jTQFBO6SXo/edit#slide=id.g269a303f9b1_0_950))
```suggestion
superchain_level: $SUPERCHAIN_LEVEL
```

Reply:
[8835e78](https://github.com/ethereum-optimism/superchain-registry/pull/140/commits/8835e7891688668ef4b1d411fde1dcd778b69d1c)